README.md

```diff
@@ -283,6 +283,16 @@ To enable tooltips you must set `enableTooltipFeature` flag to true when calling
 RInAppMessaging.configure(enableTooltipFeature: true)
 ```
 
+#### **Displaying Tooltips on tab bar buttons**
+To be able to display tooltips on UITabBar items/buttons you need to set `accessibilityIdentifier` of UITabBarItem object associated with your tab view controller. Then you need to call `updateItemIdentifiers()` method on `UITabBar` object.
+
+A code example from UIViewController's `viewDidLoad()` method
+```swift
+tabBarItem.accessibilityIdentifier = "UIElement.4"
```

Comment:
Does it mean that only `tabBarItem` will get a tooltip corresponding to `UIElement.4` identifier?

Reply:
That's right